spec/features/search/new_flow/across_england/across_england_further_education_spec.rb

```diff
@@ -0,0 +1,96 @@
+require 'rails_helper'
+
+RSpec.feature 'Searching across England' do
+  include StubbedRequests::SubjectAreas
+  include StubbedRequests::Courses
+  include StubbedRequests::Subjects
+
+  before do
+    stub_subjects
+    stub_courses_request
+    stub_subject_areas
+  end
+
+  scenario 'Candidate applies qualifications filters on results page' do
+    given_that_the_new_search_flow_feature_flag_is_enabled
+    when_i_visit_the_start_page
+    and_i_select_the_across_england_radio_button
+    and_i_click_continue
+    then_i_should_see_the_age_groups_form
+    and_the_correct_age_group_form_page_url_and_query_params_are_present
+
+    when_i_click_back
+    then_i_should_see_root_path
```

Comment:
Maybe `then_i_should_see_the_start_page` so that there is consistency with line 16?